### ISSUE TITLE
`git remote rename` does not rename *a* reference

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#534]
According to [git man](https://git-scm.com/docs/git-remote):
```
Rename the remote named <old> to <new>. 
All remote-tracking branches and configuration settings for the remote are updated.
```

book\02-git-basics\sections\remotes.asc line 196

```
If you want to rename a reference 
you can run `git remote rename` to change a remote's shortname.
```

Would it be better to delete some to?

```
You can run `git remote rename` to change a remote's shortname.
```

----

Also, at line 207

```
It's worth mentioning that this changes your remote branch names, too.
```

could be?

```
It's worth mentioning that this also changes all your remote-tracking branch names 
which are belong to that remote.
```

**Comment by ben** [comment]
Yes, except use "which belong to that remote". You're on :fire: !

**Comment by YueLinHo** [comment]
___ :fire: :fire: :fire: __________________ :fire_engine: ..... (too much???)

**Comment by ben** [comment]
Fixed with #537. :tada:
